Public/Src/Cache/ContentStore/Library/Tracing/ContentSessionTracer.cs

```diff
@@ -233,7 +233,7 @@ public virtual void PlaceFileStop(Context context, ContentHash input, PlaceFileR
         {
             if (context.IsEnabled)
             {
-                TracerOperationFinished(context, result, $"{Name}.{PlaceFileCallName} stop {result.DurationMs}ms input=[{input}] result=[{result}]");
+                TracerOperationFinished(context, result, $"{Name}.{PlaceFileCallName} stop {result.DurationMs}ms{result.GetExtraDiagnosticsMessageForTracing()} input=[{input}] result=[{result}]");

```

Comment:
>{result.GetExtraDiagnosticsMessageForTracing()} [](start = 112, length = 47)

I think this should go after the result for two reasons: 1. consistency with old message format. 2. result success is very important information and I wouldn't want that pushed aside by supplemental info.

Reply:
I decided to move this into ToString implementation.

---
In reply to: [280481495](https://github.com/Microsoft/BuildXL/pull/233#discussion_r280481495) [](ancestors = 280481495)